app/ports/management/commands/update-rsync.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
```

Comment:
`update-rsync` is a somewhat misleading name. It could be `update-portindex-and-port-data` (not the best name either, but looking for a better proposal).

Reply:
Thank you. I will change.